modules/book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(date, publisher, cover_state, archived: false, id: nil)
+    super(id, date, archived: archived)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
```

Comment:
- [ ] In the initialize method I think it would be better if the `id` could be generated automatically using `rand()` method as this would make our application to work efficiently.
- [ ]  I think it would be better if `book.rb` file could be in a folder named `classes` because `modules` refer to methods that can be included in classes. This will make reviewers easily understand your folder structure

Reply:
Hello @torobucci, 
Regarding requirement #2 The id property is coming from the parent class "Item" which has rand() so It is not required to declare it again except if I am going to change the range or something. I would like to hear from you.

Reply:
@ReemMohamedAbdelfatah 

- [ ] You are right the `id` property is generated in the `Item` parent class therefore it is not necessary for the `book` initialize method to have `id` as an argument. 
- [ ] Also super expects only two arguments `(published_date, archived)`

-    Calling `super` with the two arguments would automatically generate the id

Reply:
Done